.github/workflows/ruby-build.yml

```diff
@@ -95,6 +95,7 @@ jobs:
         uses: ./.github/actions/fetch-codeql
       - name: Build Query Pack
         run: |
+          codeql pack create ../shared/ssa --output target/packs
```

Comment:
Is this line necessary? I think the right thing would be to run run `codeql pack install ql/lib` instead. Although that probably requires `codeql/ssa` to be published previously, so there might be a bootstrapping problem.

@aeisenberg what do you think?

Reply:
The job fails if I don't add the line.

Reply:
Let's leave it in for now. I guess things go wrong at the `codeql test run` step later in the workflow, right?

Reply:
Hmm, perhaps we should actually update the `qlpack.lock.yml` so it includes the new dependency. Perhaps as a follow-up once the shared pack has been published:
```
codeql pack install ql/lib --mode=update
A fatal error occurred: Package version failure. No compatible version found: codeql/ssa@>=0.0.1 <=0.0.1.
```